aiosmtpd/tests/test_handlers.py

```diff
@@ -407,14 +407,17 @@ def test_deliver(self):
                 'anne@example.com', ['bart@example.com'], self.message)
             client.quit()
             mock().connect.assert_called_once_with('localhost', 9025)
+            # SMTP always fixes eols, so it must be always CRLF as delimeter
+            msg = NEWLINE.join([
+                'From: Anne Person <anne@example.com>',
+                'To: Bart Person <bart@example.com>',
+                'Subject: A test',
+                'X-Peer: ::1',
+                '',
+                'Testing',
+            ])
```

Comment:
This closing paren line should be indented 4 spaces.  (I should figure out why flake8 doesn't warn about that.)